src/templates/plugin.main.js

```diff
@@ -185,6 +186,12 @@ export default async (context) => {
       return false
     }
 
+    const regexpPath = getRegexpPath(localeCodes)
+    const localeMatch = route.path.match(regexpPath)
+    if (localeMatch) {
+      return localeMatch[1]
+    }
```

Comment:
Similarly to `onlyOnRoot` handling below, there is no need to do anything for `STRATEGIES.NO_PREFIX`.

Also can this code just be:
```suggestion
    if (route.path.match(getRegexpPath(localeCodes))) {
      return false
    }
```
?